lib/nerves_hub_link/configurators/cert_key.ex

```diff
@@ -1,11 +1,8 @@
-defmodule NervesHubLink.Configurator.Default do
+defmodule NervesHubLink.Configurator.CertKey do
```

Comment:
```suggestion
defmodule NervesHubLink.Configurator.CertFile do
```
Maybe we indicate this is local file based? (or some other name to indicate that we'll look for a file in a default spot)